beestation.dme

```diff
@@ -702,6 +702,7 @@
 #include "code\datums\elements\digital_camo.dm"
 #include "code\datums\elements\earhealing.dm"
 #include "code\datums\elements\embed.dm"
+#include "code\datums\elements/eyestab.dm"
```

Comment:
::'(

Reply:
Apologies, that slash wanted to be "quirky".

He will be taken our back and shot.